clash-ghc/clash-ghc.cabal

```diff
@@ -68,7 +68,7 @@ flag use-ghc-paths
 executable clash
   Main-Is:            src-ghc/Batch.hs
   Build-Depends:      base, clash-ghc
-  GHC-Options:        -Wall -Wcompat -threaded -rtsopts
+  GHC-Options:        -Wall -Wcompat -threaded -rtsopts -with-rtsopts=-A128m
```

Comment:
No love for `clashi`? :)

Reply:
Hmm good point. I would guess it doesn't normally matter for `clashi` until you do something like `:vhdl` which kicks of normalization etc.

Reply:
So it should probably have it for `clashi` too I guess

Reply:
If it's not tons of work, maybe just throw these things into a `common` stanza, seeing how we probably want them to be the same for `clash` and `clashi`

Reply:
And we can just `import: rts-config`